src/coreclr/jit/gentree.h

```diff
@@ -559,7 +559,8 @@ enum GenTreeFlags : unsigned int
     GTF_ICON_CIDMID_HDL         = 0x0E000000, // GT_CNS_INT -- constant is a class ID or a module ID
     GTF_ICON_BBC_PTR            = 0x0F000000, // GT_CNS_INT -- constant is a basic block count pointer
     GTF_ICON_STATIC_BOX_PTR     = 0x10000000, // GT_CNS_INT -- constant is an address of the box for a STATIC_IN_HEAP field
-    GTF_ICON_FIELD_SEQ          = 0x11000000, // <--------> -- constant is a FieldSeq* (used only as VNHandle)
+    GTF_ICON_FIELD_SEQ          = 0x11000000, // GT_CNS_INT -- constant is a FieldSeq* (used only as VNHandle)
+    GTF_ICON_STATIC_ADDR_PTR    = 0x13000000, // GT_CNS_INT -- constant is a pointer to a static field address
```

Comment:
Why `0x13000000` and not `0x12000000`?

Reply:
> Why `0x13000000` and not `0x12000000`?

because `GTF_ICON_OBJ_HDL` is `0x12000000` and I added it near to `GTF_ICON_STR_HDL` for readability